client/app/components/Input/InputTextarea.jsx

```diff
@@ -67,84 +67,75 @@ export type State = {
   value?: any,
 };
 
-export class InputTextarea extends React.Component<Props, State> {
-  editor: any;
-
-  editorRef: any;
-
-  constructor(props: Props) {
-    super(props);
-    this.state = {
-      value: props.value || '',
-    };
-  }
-
-  componentDidMount() {
-    const { value } = this.state;
-    if (this.editorRef) {
-      this.editor = init({
-        element: this.editorRef.getElementsByClassName('editor')[0],
-        onChange: this.onChange,
-        classes,
-        actions,
-      });
-      this.editor.content.innerHTML = value;
-    }
-  }
-
-  onChange = (value: string) => {
-    this.setState({ value });
+export function InputTextarea({
+  id,
+  name,
+  value,
+  required,
+  hasError,
+  myRef,
+  dark,
+}: Props) {
+  const [currentValue, setValue] = useState<State>(value || '');
+  let editorRef;
```

Comment:
Rather than use a callback ref, `useRef()` can be used instead:

```
  const editorRef = useRef(null);
  ...
  return (
    <div
      id={id}
      ...
      ref={editorRef}
    >
      ...
    </div>
  );
```

Then references to `editorRef` would be replaced with `editorRef.current`, e.g. inside `onFocus`:

```
  if (editorRef.current) {
    editorRef.current.getElementsByClassName('editorContent')[0].focus();
  }
```